src/app/app.component.ts

```diff
@@ -305,4 +316,27 @@ export class AppComponent implements OnDestroy {
       }
     }
   }
+
+  /**
+   * since page load and animation time are not always equal
+   * an interval seemed to feel the most responsive
+   */
+  private _focusElement(id: string) {
+    let counter = 0;
+    this._intervalTimer = setInterval(() => {
+      counter += 1;
+
+      const el = document.getElementById(`t-${id}`);
+      el?.focus();
+
+      if (el && IS_MOBILE) {
+        el.classList.add('mobile-highlight');
+        el.addEventListener('blur', () => el.classList.remove('mobile-highlight'));
+      }
```

Comment:
Since we don't want focus to be visible on mobile I added this class to make it visible just after searching.